pkg/packagekit/updates.jsx

```diff
@@ -744,24 +776,47 @@ class OsUpdates extends React.Component {
                     return <div className="spinner spinner-lg progress-main-view" />;
 
             case "available":
+                // when unregistered, hide the Apply buttons and show a warning
+                if (this.state.unregistered) {
+                    unregisteredWarning = (<div>
+                        <h2>{ _("Unregistered System") }</h2>
+                        <div className="alert alert-warning">
+                            <span className="pficon pficon-warning-triangle-o"></span>
+                            <span>
+                                <strong>{ _("Updates are disabled.") }</strong>
+                                &nbsp;
+                                { _("You need to re-subscribe this system.") }
+                            </span>
+                            <button className="btn btn-primary pull-right"
+                                    onClick={ () => cockpit.jump("/subscriptions") }>
+                                { _("View Registration Details") }
+                            </button>
+                        </div>
+                    </div>);
+                } else {
+                    applyAll = (<button className="btn btn-primary"
```

Comment:
I suggest (here and in other places) to start the opening tag on the next line with a default indentation.